.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  main:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
The build engine being used is Cake, so it's best to run the cake scripts instead ("buildandtest" target).